### PR TITLE
docs: update oct prerelease changelog

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: packages/vscode-extension/PRERELEASE.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 ## Changelog
 
-### September 26, 2023
+### October 10, 2023
 
 #### New Features
 
+- **API-Based Message Extension**: We have introduced a whole new way to build search-based message extension - from a new API with Azure Functions or from your existing API with an OpenAPI description document. This is a great starting point for building a message extension that requires a backend to fetch data from a third-party API or a database.
```
Comment (sffamily) [inline packages/vscode-extension/PRERELEASE.md:9]: Shall we make it more clear that this release is supporting read-only API or GET only? as the last sentence is indicating so?

File: packages/vscode-extension/PRERELEASE.md [hunk 2]
```diff
@@ -2,15 +2,18 @@
...
     ![Sample Gallery](https://github.com/OfficeDev/TeamsFx/assets/11220663/87a705e2-aa79-46dc-87ba-204f20fd3771)
 
-- **Custom Search Results Template Optimization**: We've optimized the `Custom Search Results` project template to seamlessly integrate with Microsoft 365 Copilot. To utilize this feature, please enable the `Develop Copilot Plugin` feature setting via Visual Studio Code in the [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) and enroll in the Microsoft 365 Copilot [Early Access Program](https://aka.ms/PluginsEarlyAccess).
-    ![Copilot Plugin](https://github.com/OfficeDev/TeamsFx/assets/11220663/7fba4a92-f912-4021-b048-9f72de1267d8)
+- **Custom Search Results Template Optimization**: We've optimized the `Custom Search Results` (Using Bot Framework) project template to seamlessly integrate with Microsoft 365 Copilot. To utilize this feature, please enable the `Develop Copilot Plugin` feature setting via Visual Studio Code in the [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) and enroll in the Microsoft 365 Copilot [Early Access Program](https://aka.ms/PluginsEarlyAccess).
```
Comment (sffamily) [inline packages/vscode-extension/PRERELEASE.md:15]: Currently to try it, one may need enrolling in that EAP, turn on feature flag, and also need their Dev Tenant to be whitelisted? Not sure how best describe this? or pass the explanation to some other links?

File: packages/vscode-extension/PRERELEASE.md (hunk 1 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:9]: Our UI indicates only `GET` API is supported, also there's going to be a blog announcement for API-ME so I didn't cover it in Changelog.

File: packages/vscode-extension/PRERELEASE.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/PRERELEASE.md:15]: We will mention this in a blog, seems too verbose to add all these information in changelog

File: packages/vscode-extension/PRERELEASE.md (hunk 1 above)
Comment (sffamily) [reply packages/vscode-extension/PRERELEASE.md:9]: that's ok as long as it is covered and clear on UI.

File: packages/vscode-extension/PRERELEASE.md (hunk 2 above)
Comment (sffamily) [reply packages/vscode-extension/PRERELEASE.md:15]: blog works. more people will read blogs than changelogs.

**Review by sffamily — APPROVED** [review]

**Pull request merged**